### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by charliec404** [charliec404/cc-wizbiz-website#20]
Alt text, labels, keyboard order.

**Comment by charliec404** [comment]
Not Ready: Missing priority, assignee, milestone, and size estimate. The specific accessibility improvements required and how they will be tested need to be identified.
